test/Libraries/SystemTestServices/SystemTestBase.cs

```diff
@@ -383,6 +385,17 @@ public void AssertClassName(string guid, string className)
             Assert.AreEqual(classInfo.ClassName, className);
         }
 
+        /// <summary>
+        /// Get loaded instance(s) of a view extension type.
+        /// Returns null if no view extensions of the provided type are loaded
+        /// </summary>
+        /// <param name="type">A type which implements IViewExtension</param>
+        /// <returns></returns>
+        public IEnumerable<IViewExtension> GetViewExtensionsByType(Type type)
+        {
+            return View.viewExtensionManager.ViewExtensions.Where(v => v.GetType() == type);
```

Comment:
you can use `<TypeOf>`

Reply:
@mjkkirschner I have updated this to be a generic method. Thanks for your help.